config.js

```diff
@@ -60,10 +60,11 @@ const CACHE_EXPIRY_IN_SECS = pe.CACHE_EXPIRY_IN_SECS ||
   DEFAULT_CACHE_EXPIRY_IN_SECS;
 
 // request limiter settings
-const rateLimit = pe.DDOS_RATE_LIMIT;
-const rateWindow = pe.DDOS_RATE_WINDOW;
-const endpointToLimit = pe.DDOS_ENDPOINT_TO_LIMIT;
-const httpMethodToLimit = pe.DDOS_HTTP_METHOD_TO_LIMIT;
+const expressLimiterPath = pe.EXPRESS_LIMITER_PATH;
+const expressLimiterMethod = pe.EXPRESS_LIMITER_METHOD;
+const expressLimiterLookup = pe.EXPRESS_LIMITER_LOOKUP || 'headers.UserName';
```

Comment:
I think this one needs to JSON parse the string, in case the user provides an array of strings here